src/components/Disturber.tsx

```diff
@@ -78,69 +79,76 @@ export const Disturber = ({ navigation, publicJsonFile }: Props) => {
 
   if (!isVisible || !closestItem) return null;
 
+  const {
+    backgroundColor,
+    button,
+    description,
+    headline,
+    picture,
+    showButtonToClose = true,
+    title
+  } = closestItem;
+  const { title: buttonTitle, navigationTo, params } = button;
+
+  const showButton = !!button && !!buttonTitle && !!navigationTo && !!params;
```

Comment:
if button was successfully destructed, it wouldn't be needed to check again.
i suggest removing the destruction and check every key of `button.` here and use `button.` as well in the render part?